python/lsst/cp/pipe/measureCrosstalk.py

```diff
@@ -109,6 +110,10 @@ class CrosstalkExtractConfig(pipeBase.PipelineTaskConfig,
         default=True,
         doc="Is the input exposure trimmed?"
     )
+    background = ConfigurableField(
+        target=SubtractBackgroundTask,
+        doc="Configuration for initial background estimation",
```

Comment:
This isn't quite a configuration in the sense of a set of parameters. May be a better doc would be "Initial background estimation task"?